src/generic/NetworkManager.hpp

```diff
@@ -23,6 +23,13 @@ class NetworkManager {
   // Public for testing
   static const uint8_t kRecentIdsCacheSize = 5;
 
+  // If we've seen this many copies for the packets in our recent ids cache,
+  // then stop rebroadcasting for a while, to avoid flooding the network.
+  static constexpr uint16_t kPacketCopiesThreshold = 10 * kRecentIdsCacheSize;
```

Comment:
Are consts not UPPER_SNAKE_CASE?

Reply:
Nope: https://google.github.io/styleguide/cppguide.html#Constant_Names

Enum names can either be `kEnumName` or `ENUM_NAME`.